analyzer/consensus/consensus.go

```diff
@@ -316,6 +318,10 @@ func (m *processor) queueDbUpdates(batch *storage.QueryBatch, data allData) erro
 		}
 	}
 
+	if m.mode == analyzer.SlowSyncMode {
+		batch.Queue(validatorUptimesViewRefreshQuery)
```

Comment:
Let's refresh this in a new `ItemBasedAnalyzer`, see `accounts_list.go` for an example. Alternatively, feel free to refactor that analyzer to refresh both views; I think we'll soon have more views that need refreshing, and it'd be good to have them all in one spot so we can tweak how often they get recalculated.